contracts/interfaces/IStakingViewer.sol

```diff
@@ -93,6 +94,8 @@ interface IStakingViewer {
 
   function getManagerTokenRewards(address manager) external view returns (Token[] memory tokens);
 
+  function getManagerTotalRewards(address member) external view returns (uint managerTotalRewards);
+
   function getManagerPoolsAndRewards(address manager) external view returns (ManagerPoolsAndRewards memory);
 
   function getManagerRewards(uint[] memory poolIds) external view returns (Token[] memory tokens);
```

Comment:
1. please stay consistent with naming the function argument: either `manager` or `member` across these functions
2. Slight scope creep: `getManagerTokenRewards` & `getManagerRewards` could be renamed to something a bit more descriptive: `getManagerTokenRewardsByAddr`, `getManagerTokenRewardsByPools`,

Reply:
i'll change to `getManagerTokenRewardsByAddr` but we'll need to keep `getManagerRewards` to avoid a breaking change as its an existing method